hydra_base/lib/storage/hdfstorageadapter.py

```diff
@@ -74,7 +92,7 @@ def get_hdf_config(config_key="storage_hdf", **kwargs):
     @filestore_url("url")
     def open_hdf_url(self, url, **kwargs):
         try:
-            with fsspec.open(url, mode='rb', anon=True, default_fill_cache=True) as fp:
+            with fsspec.open(url, mode='rb', anon=self._get_anon(), default_fill_cache=True) as fp:
```

Comment:
As the kwargs to `fsspec.open` are used several times, they can be put in a dict on the class (or prperty that returns a dict) and reused in each `fsspec.open` call, e.g.:

```python
  fsspec_args = {
    "mode": "rb",
    ...etc...
  }
  fsspec.open(url, **fsspec_args)
```

Reply:
done